WordPress/Classes/Utility/BuildInformation/FeatureFlag.swift

```diff
@@ -96,6 +99,8 @@ extension FeatureFlag {
             return "Recommend App to Others"
         case .newCommentEdit:
             return "New Comment Edit"
+        case .domains:
+            return "Domains selling"
```

Comment:
I believe this is internally visible only, and if that's the case this is trivial and can be ignored.

Isn't the feature "Domain Purchases" from the PoV of the user?

Reply:
Yes it is!! Thanks for spotting this